logback-classic/src/main/java/ch/qos/logback/classic/spi/ThrowableProxyUtil.java

```diff
@@ -179,6 +187,11 @@ public static void subjoinFirstLineRootCauseFirst(StringBuilder buf, IThrowableP
     }
 
     private static void subjoinExceptionMessage(StringBuilder buf, IThrowableProxy tp) {
-        buf.append(tp.getClassName()).append(": ").append(tp.getMessage());
+        //The JVM does not print the message if it is null
```

Comment:
How is this related to the main purpose of this PR? Are you fixing some other issue here as well?

Reply:
You are right, it has nothing to do with the main purpose of the PR.  I saw another issue as I created the PR related to a test case where when the message was null, logback would print "null".  The JRE would not print "null", and looking at the test cases, the intent of logback with regard to exceptions was to mimic what the JRE did.